EIPS/eip-draft-returndata.md

```diff
@@ -0,0 +1,31 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: Embedding transaction return data in receipts
+    Author: Nick Johnson <nick@ethereum.org>
+    Type: Standard Track
+    Category Core
+    Status: Draft
+    Created: 2017-06-30
+    Requires: 140
+    Replaces: 98
+
+
+## Abstract
+This EIP replaces the intermediate state root field of the receipt with a status code indicating if the top-level call succeeded or failed.
+
+## Motivation
+With the introduction of the REVERT opcode in EIP140, it is no longer possible for users to assume that a transaction failed iff it consumed all gas. As a result, there is no clear mechanism for callers to determine whether a transaction succeeded and the state changes contained in it were applied.
```

Comment:
typo: iff

Reply:
I believe it's intended ("if and only if")

Reply:
[Here's a transaction](https://etherscan.io/tx/0x842e89d44f1de6cd26bca31d492932bfb4222261c08918bd7fcc39074b22fc2d) that consumed all gas and didn't fail.

It was never possible to make the assumption to begin with. ;)

Reply:
It wasn't, but people still did.